framework/mesh/mesh_continuum/mesh_continuum.cc

```diff
@@ -314,7 +313,7 @@ MeshContinuum::FindAssociatedCellVertices(const CellFace& cur_face,
       log.LogAllError() << "Face DOF mapping failed in call to "
                         << "MeshContinuum::FindAssociatedVertices. Could not find a matching"
                            "node."
-                        << cur_face.neighbor_id << " " << cur_face.centroid.PrintS();
+                        << cur_face.neighbor_id << " " << cur_face.centroid.PrintStr();
       Exit(EXIT_FAILURE);
```

Comment:
Instead of `Exit`, we should throw an exception.

Reply:
done